src/main/scala/beam/utils/google_routes_db/sql/package.scala

```diff
@@ -0,0 +1,21 @@
+package beam.utils.google_routes_db
+
+import com.google.maps.model.LatLng
+
+package object sql {
+
+  type GeometryPoint = String
+  type GeometryLinestring = String
+
+  val projection: Int = 4326
```

Comment:
What is this really? Can we make the naming more descriptive?

Reply:
I'm still not quite sure the new name is much better, but maybe I am not familiar enough with the underlying names of this domain?